contracts/tokens/BodhiToken.sol

```diff
@@ -18,11 +18,6 @@ contract BodhiToken is StandardToken, Ownable {
     function BodhiToken() public {
     }
 
-    /// @dev Fallback function that throws.
-    function() external {
-        assert(false);
-    }
-
```

Comment:
I fine to remove this. However, when we deploy this on Qtum, we will have a potential issue that some users might directly/mistakenly sent QTUM to this address, and the QTUM will be totally lost because there is no way to return it back to users.

Reply:
From the docs, it says a contract can't receive payment unless there is a `fallback` function that is marked `payable`. (http://solidity.readthedocs.io/en/develop/contracts.html#fallback-function)

Contracts should throw if no `payable fallback function` is present and ether (or should be in our case, QTUM).

Does this function differently on the QTUM blockchain?